### PR TITLE
[Themes-1688] Fixed inconsistent image width

### PR DESCRIPTION
**Pull request by malavikakoppula**
#### Jira Ticket: [THEMES-1688](https://arcpublishing.atlassian.net/browse/THEMES-1688)

- Headlines in small and medium promos on mobile take up 66% container width
- Width of images in small and medium promos on mobile are the same in top table list

## Test Steps

1. Checkout this branch `git checkout THEMES-1688`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/top-table-list-block`
3. Add medium and small promos and then inspect to check the width

[THEMES-1688]: https://arcpublishing.atlassian.net/browse/THEMES-1688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .storybook/themes/news.scss [hunk 1]
```diff
@@ -3266,7 +3270,17 @@
...
 							"clear": both,
-							"margin-block-start": var(--global-spacing-5)
+							"margin-block-start": var(--global-spacing-5),
+						),
+						"heading": (
+							"font-size": var(--heading-level-5-font-size),
+							"font-weight": var(--heading-level-5-font-weight),
+							"line-height": var(--heading-level-5-line-height),
+							"margin-block-end": var(--global-spacing-4),
+							"width": 66%
```
Comment (nschubach) [inline .storybook/themes/news.scss]: `inline-size`

File: blocks/top-table-list-block/themes/news.json [hunk 2]
```diff
@@ -158,7 +158,8 @@
 						"font-size": "var(--heading-level-5-font-size)",
 						"font-weight": "var(--heading-level-5-font-weight)",
 						"line-height": "var(--heading-level-5-line-height)",
-						"margin-block-end": "var(--global-spacing-4)"
+						"margin-block-end": "var(--global-spacing-4)",
+						"width": "66%"
```
Comment (nschubach) [inline blocks/top-table-list-block/themes/news.json]: `inline-size`

File: .storybook/themes/news.scss (hunk 1 above)
Comment (malavikakoppula) [reply .storybook/themes/news.scss]: @nschubach Fixed it

File: blocks/top-table-list-block/themes/news.json (hunk 2 above)
Comment (malavikakoppula) [reply blocks/top-table-list-block/themes/news.json]: @nschubach Fixed it

**Comment by nschubach** [comment]
I'm getting an issue where the header overflows into the margins: 
image
it can be fixed by setting the inline-size to `calc(66% - var(--global-spacing-6))` (to match the margin assigned in the desktop/components/heading)

**Comment by nschubach** [comment]
that should only be applied on the desktop though since the mobile would indent too far: (with)
image

(without inline-size mobile / allow the float to space it)
image

**Review by nschubach — APPROVED** [review]
Not sure what I was seeing with the headings, it seems to be ok.

**Pull request merged**